### ISSUE TITLE
Featured web desktops

### ISSUE THREAD

**Issue by SMUsamaShah** [syxanash/awesome-web-desktops#65]
Hi, this collection is great and I have been watching for all updates and usually checkout every new addition. 

Some time ago I went through *all* of the web desktops in the list. What I found was that only a handful of these are high quality and worth checking out. My top picks are windows93 and windows96. Rest of them are mostly just non functional UI mockups featuring a few movable widgets/windows/dualogs and nothing else. 

I think you should consider adding a new "Featured" section listing your own handful favourites. This might also encourage people to submit higher quality desktops or to polish them a bit more.

**Comment by SMUsamaShah** [comment]
Going through the list randomly. 

- Don't know what makes its a web desktop other than few stylized buttons http://cyberspaceandtime.com/Gaano9Y6KAU.video+related
- some moveable widgets but not all https://wwwtxt.org/tagged/img https://wwwtxt.org/about
- some random picks https://erd-os.iamerd.xyz/ https://wuerfeldev.de/imprint https://jackadam.cc/?Statement https://thecodingforge.com.au/ https://www.browso.app/
- not even movable windows, most icons buttons are non-funct https://56k-modem.online/
- also https://violet.pm/

There are loads of unique things in here. Most are experimental already. windows93/96 are my picks but if you go through. There are many functional and unique ones.

**Comment by syxanash** [comment]
oh I see your point, and what would be your definition of functional ones? (such as windows93/96) like what kind of functionality list would qualify these for a Featured web desktop?

I'm just trying to understand your point of view a bit more and see if I could create different sub categories.

**Comment by SMUsamaShah** [comment]
While picking these out for you I was rethinking what is considered as a web desktop. Looking at these it felt like if a UI has at least a feeling (not functionality necessarily) that it was part of a desktop OS, it will qualify as web desktop. IMO, the first link above fits that criteria.

Picking up from your response "sub categories" is a much better idea. Some of these try to simulate faithful and full desktop experience e.g. Windows93 (there are few others). Some are portfolios or CVs where a dev showed off they can make floating widgets and a desktop with open able icons etc. Some are unique experiences like the  restart one which simulates restarting of various OSes. There are some musical ones. Some might go into a misc category. 

There can be a few groups overall. And it actually might be more encouraging than just the featured or experimental category.

From the links I gave as examples, do you think they are all web desktops and just need better grouping?

**Comment by syxanash** [comment]
You're very right, a lot of websites have a web desktops designs in principle but have lots of UI mockups without functionality. I still add them to the list because of the creativity and other experiences they might give the user.

I can list a few actually, [The Restart](http://www.therestartpage.com/), [Windows 1.0](https://win1.krnl386.com/), [glenda.](https://glenda.0x46.net/), while these have a web desktop metaphor as a base they might lack that full experience you mentioned earlier. Let me see if with a bit of brainstorming we could group these better.

**Comment by DustinBrett** [comment]
> Hi, this collection is great and I have been watching for all updates and usually checkout every new addition.
> 
> Some time ago I went through _all_ of the web desktops in the list. What I found was that only a handful of these are high quality and worth checking out. My top picks are windows93 and windows96. Rest of them are mostly just non functional UI mockups featuring a few movable widgets/windows/dualogs and nothing else.
> 
> I think you should consider adding a new "Featured" section listing your own handful favourites. This might also encourage people to submit higher quality desktops or to polish them a bit more.

I agree that it would be cool to have a featured section and I think that section could probably have 10-20+ from what I have seen from this list.

I am curious @SMUsamaShah what you thought of my entry ([daedalOS](https://github.com/DustinBrett/daedalOS)) compared to your top picks windows 93 & 96, which were inspirations of mine when it came to realism. I am always striving to improve my project and would love any criticisms you have about it or how I could improve it to make it one of your top picks. Thanks!

**Comment by SMUsamaShah** [comment]
@DustinBrett 

It looks great and you have put a lot of work into it. Looking at the repo, it has a lot to offer even for practical use.

Windows93 and 96 make all their apps etc discoverable directly by putting them all on desktop. I think you should try the same by default and offer a cleaner desktop on a /#clean url perhaps. If I just open your site on the first look it appears there isn't much in there (while it offers a lot in fact). I will definitely put it among featured ones. Windows 98 was my first OS so there is that nostalgic connection too with 93 and 96. 

P.S. Since you as author of this desktop is asking me (a third person who is merely and audience of the list) , I feel weird/bad/humbled being asked to give an opinion about it. Your desktop is great, absolutely not one of those quick mockups which most are.

There can be a featured/personal favourites section on top with ~10-20 sites. There can be another section called 'pick(s) of the day' which features a site randomly (never the same site twice). Below can be rest of the sites all in random order.

**Comment by DustinBrett** [comment]
> @DustinBrett
> 
> It looks great and you have put a lot of work into it. Looking at the repo, it has a lot to offer even for practical use.
> 
> Windows93 and 96 make all their apps etc discoverable directly by putting them all on desktop. I think you should try the same by default and offer a cleaner desktop on a /#clean url perhaps. If I just open your site on the first look it appears there isn't much in there (while it offers a lot in fact). I will definitely put it among featured ones. Windows 98 was my first OS so there is that nostalgic connection too with 93 and 96.
> 
> P.S. Since you as author of this desktop is asking me (a third person who is merely and audience of the list) , I feel weird/bad/humbled being asked to give an opinion about it. Your desktop is great, absolutely not one of those quick mockups which most are.
> 
> There can be a featured/personal favourites section on top with ~10-20 sites. There can be another section called 'pick(s) of the day' which features a site randomly (never the same site twice). Below can be rest of the sites all in random order.

Thank you very much for checking it out and the suggestions. That's a good point about my apps being kind of hidden as I have gone for a minimalistic desktop look which matches my real personal desktop.

I appreciate your feedback a lot and will think on ways I can tweak my project to make it's features more visible/obvious.

The 'pick of the day idea' is cool too. I'd posted a while ago (https://github.com/syxanash/awesome-web-desktops/issues/47) that I think this could be a community of web desktop builders and that sounds like a good addition.

The main site https://simone.computer/#/webdesktops has a cool interface and I think it could be tweaked to build upon the great list that is already here.

**Comment by syxanash** [comment]
Hey folks, thank you so much for the feedback, this was a really interesting discussion. I got great point of views from your messages. For the moment, I think I'll keep this as a single list without other sub categories. The original idea was just to randomly get inspired by discovering new sites around the web and look up the technologies used in some of these projects to make your own. Maybe in the coming months I'll look into sub categories work a bit more.

**Comment by syxanash** [comment]
Hi folks 👋 
I decided to take on this task again and implemented a first iteration of subcategories for AWB. You can access the subcategories view through my website. If you'd like to have a look, let me know what you think :)
Thanks again for your valuable feedback during the previous conversation!

https://simone.computer/#/webdesktops 

<img width="658" alt="Screenshot 2025-01-13 at 15 27 33" src="https://github.com/user-attachments/assets/44c8d149-7e64-424a-8daf-24f74e37f696" />
